saleor/celery_mail.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import unicode_literals
+
+import os
+from celery import Celery
+from django.conf import settings
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "saleor.settings")
+
+app = Celery('mail')
```

Comment:
Why do we use name "mail" here (and "celery_mail" as filename)? Is it because it's intended to be used for sending emails only? I thought we have one configuration multiple kinds of tasks, so the name should be more general.